### ISSUE TITLE
Resume from last

### ISSUE THREAD

**Issue by MrConn** [joeyberkovitz/plex-aaos#1]
The app doesn't remember where it was on a given track.
I'm using the app to listen to a 19 hour audio book, and without a resume or bookmark-function it's kinda troublesome.
But I love that someone is working on this app! 😁

**Comment by joeyberkovitz** [comment]
appreciate the feedback, I'll look into adding this feature
I'm already tracking the last played song upon any player state change (https://github.com/joeyberkovitz/plex-aaos/blob/master/automotive/src/main/java/us/berkovitz/plexaaos/MyMusicService.kt#L624), so I can probably adjust that same code to store the position in the song as well.

**Comment by MrConn** [comment]
Awesome!
Looking forward to the update 👍😊

Få Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by micksmothers** [comment]
FYI, is tracking the last played song also based on playlist? I suspect this is in the next version as currently it will remember my playlist, but starts with the first song every time it restarts.

**Comment by joeyberkovitz** [comment]
It should remember both the playlist and song, but maybe a data loading issue is causing the playlist/song index to be thrown off.

**Comment by MrConn** [comment]
For me, my audio book starts from the beginning every time. I only have one playlist, and that playlist contains one 'song' (the audio book).

Få Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by joeyberkovitz** [comment]
It's been a while, but I finally got some time to work on this
version 0.4.4 is out for review by Google and adds support for remembering the position in the last played song. I also fixed the bug where it always reverts to the first song in the last played playlist. loading should also improve a bit, but there may still be some issues if the app is opened (or the car requests info) before an internet connection is available.
